src/components/Discord.tsx

```
@@ -1,9 +1,107 @@
+                    throw new Error("HTTP ERROR");
+                }
+                const data = await response.json();
+
+                if (data.status === "COMPLETE") {
+                    setIsDiscordInvitedSuccess(true);
+                    setIsChecking(false);
+                    return;
+                }
+                attempts++;
+                setTimeout(poll, interval);
+            } catch (err: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)